### ISSUE TITLE
Conditionals improvements

### ISSUE THREAD

**Issue by Toxicom** [DelvUI/DelvCD#12]
# Description
I believe the conditionals window can be improved and I hope my suggestions will be considered

## 1) Multiple triggers for conditional styling
As it stands right now, there's no way to check for 2 conditions before applying icon changes. I imagine the options UI would have to change where you create a condition and then inside select and/or/xor triggers so that you can have multiple.

### Example use case
Best example I can think of right now is Reaper's **Gibbet** & **Gallows** abilities. For those two abilities, I need to check for:
1. Soul Reaver is active (Trigger 1)
2. Enhanced Gibbet/Gallows is active (Trigger 2)
![image](https://github.com/DelvUI/DelvCD/assets/69549795/2cea011f-ac6a-4157-b500-3e3d33a16223)

So in a conditionals window, I imagine the structure as follows:
- Conditional 1
   - If Trigger 1 is active AND Trigger 2 is active apply styling
   - If either of triggers is false, move to Condition 2
- Conditional 2
   - If Trigger 1 is active apply styling

###  Notes
It would also make sense to have the option to give custom names for conditionals, similarly as you can for groups/icons/bars

## 2) Optional override for conditional styling
I've seen this done in a different Auras plugin, unfortunately it was closed source and no longer maintained. What I mean by optional overrides is basically introducing a second checkbox next to the existing checkboxes in the current conditionals window. Example mock below:
![image](https://github.com/DelvUI/DelvCD/assets/69549795/50b02888-c37d-4030-9159-eb6d9594f5de)

Basically what this does is introduces inheritance from previous conditions and isolates the styling to that specific conditions, so if on the root I have set these parameters
- Glow segments 4
- Border Thickness 3

Then for this specific condition, it would inherit all the styles from the root icon and additionally apply
- Glow
- Glow Segments 6
- Anim. speed 0.5
- Glow Color

### Example use case
Here I can use the same Gibbet & Gallows example, where on Condition 1 I could set Desaturate as disabled and then on Condition 2 apply glow while inheriting the already disabled Desaturate if the Condition 1 also passed.

## Ending notes
I'm terrible at writing so excuse me if it's not clear, let me know and I'll try to elaborate better
